quetz/tasks/mirror.py

```diff
@@ -114,6 +114,8 @@ def __init__(self, host: str, path: str, session=None):
         self.file = SpooledTemporaryFile()
         response.raw.decode_content = True  # for gzipped response content
         shutil.copyfileobj(response.raw, self.file)
+        response.close()
+        session.close()
```

Comment:
argh, this should only be closed if the session didn't come as parameter. Can you fix this?

Reply:
https://github.com/mamba-org/quetz/pull/347